### DISCUSSION TITLE
Should this be a bug

### DISCUSSION THREAD

**Discussion by nalexiou-loc** [NiklasMerz/cordova-plugin-fingerprint-aio#350] [Q&A] [answered]
User sees an overlay, on samsung s21.  

Verify your identity, and of course the plugin popup behind that.  

See screen. [https://ibb.co/mTfd6mF](https://ibb.co/mTfd6mF)

Not sure if this is a bug, although the user thinks so.  Thoughts?

**Comment by NiklasMerz** [comment]
What happens when you use your fingerprint sensor? Maybe that's some weird Samsung UI.

**Reply by nalexiou-loc** [reply]
Sensor is on the screen, google "Samsung s21" videos.  Weird, yes. Additionally, you could display an icon, or an animated icon.     
Yeah seems like it is required. According to some programmers on reddit.  (not a good source)  https://www.reddit.com/r/galaxys10/comments/bgfosh/is_there_any_way_to_disable_the_fullscreen_verify/

Would be best if we showed only one popup or overlay. Not both.

**Comment by NiklasMerz** [comment, marked as answer]
I think we can do nothing about this as Samsung chose to built the UI like this and we just use the native API for this plugin.
